t/t2800-jobs-cmd.t

```diff
@@ -571,6 +571,20 @@ test_expect_success 'flux-jobs --format={runtime:0.3f} works' '
 	[ "$(grep -E "\.[0-9]{3}" runtime-dotRI.out | wc -l)" = "15" ]
 '
 
+test_expect_success 'flux-jobs --format={contextual_time} works' '
+	flux jobs --filter=pending -c1 -no "{contextual_time:h}" \
+		>ctx_timeP.out &&
+	flux jobs --filter=running -c1 -no "{contextual_time:h}" \
+		>ctx_timeR.out &&
+	flux jobs --filter=completed -c1 -no "{contextual_time:h}" \
+		>ctx_timeCD.out &&
+	echo "300.0" >duration.expected &&
+	test_cmp duration.expected ctx_timeP.out &&
+	test_must_fail test_cmp duration.expected ctx_timeR.out &&
+	test_must_fail test_cmp duration.expected ctx_timeCD.out &&
+	grep -E "\.[0-9]{3}" ctx_timeCD.out
+'
```

Comment:
perhaps also add a test for some failed job too.  Off the top of my head, a canceled pending job should have runtime = 0.  The timed out job should have a runtime > 0 as well.  I can't remember if a `nosuchcommand` job failure hits the running state or not, but that's another one to try.

I notice that such tests were not added to the `{runtime}` tests above this, so dunno if worth copying over there.  B/c of `contextual_time`'s logic, want to make sure inactive jobs are picking up the runtime instead of the duration.

Reply:
Yeah, the test already demonstrates that `{runtime}` is used for an inactive job and I had figured the `{runtime}` rendering logic was already tested elsewhere, so seemed like it would be a redundant test.